src/types/database.ts

```diff
@@ -28,6 +34,40 @@ export interface Database {
   }
   public: {
     Tables: {
+      attendee: {
+        Row: {
+          attending: Database["public"]["Enums"]["attending"]
+          event_id: string
+          remark: string | null
+          user_id: string
+        }
+        Insert: {
+          attending: Database["public"]["Enums"]["attending"]
+          event_id: string
+          remark?: string | null
+          user_id: string
+        }
+        Update: {
+          attending?: Database["public"]["Enums"]["attending"]
+          event_id?: string
+          remark?: string | null
+          user_id?: string
+        }
+        Relationships: [
```

Comment:
Why do you need these? Other table relationships haven't been defined like this

Reply:
this is autogenerated